noir-projects/aztec-nr/aztec/src/oracle/arguments.nr

```diff
@@ -1,11 +1,24 @@
+#[oracle(packArgumentsArray)]
+fn pack_arguments_array_oracle<N>(_args: [Field; N]) -> Field {}
+
 #[oracle(packArguments)]
-fn pack_arguments_oracle<N>(_args: [Field; N]) -> Field {}
+fn pack_arguments_oracle<N>(_args: [Field]) -> Field {}
```

Comment:
Does it still need the `N` type parameter if it's a slice? (purely asking, I haven't used Noir's slices yet)

Reply:
Right, it's not needed!